manifests/s/SpanishGovernment/InstaladorDNIe/1.1.0/SpanishGovernment.InstaladorDNIe.locale.es-ES.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: SpanishGovernment.InstaladorDNIe
+PackageVersion: 1.1.0
+PackageLocale: es-ES
+Publisher: Cuerpo Nacional de Policía
+PublisherUrl: https://www.dnielectronico.es/PortalDNIe/
+Author: Spanish Goverment
+PackageName: Instalador Tarjetas DNIe
+PackageUrl: https://www.dnielectronico.es/PortalDNIe/PRF1_Cons02.action?pag=REF_1101&id_menu=66
+License: Freeware
+Copyright: Copyright (c) 2020 Flexera. All Rights Reserved.
+ShortDescription: This driver allows the DNIe to operate as a plug-and-play device on Windows systems, automatically installing the necessary software for its use once inserted into a compatible card reader.
+Description: This driver allows the DNIe to operate as a plug-and-play device on Windows systems, automatically installing the necessary software for its use once inserted into a compatible card reader.

```

Hi @antonioalopezfernandez,

Should the `Description` and `ShortDescription` in the `es-ES` localization files be in Spanish? Otherwise this PR appears to be ready to merge. 

Internet translation - feel free to improve. 
```suggestion
ShortDescription: Este controlador permite que el DNIe funcione como un dispositivo plug-and-play en sistemas Windows, instalando automáticamente el software necesario para su uso una vez insertado en un lector de tarjetas compatible.
Description: Este controlador permite que el DNIe funcione como un dispositivo plug-and-play en sistemas Windows, instalando automáticamente el software necesario para su uso una vez insertado en un lector de tarjetas compatible.
```

---

Hello,

I think that the target audience is going to be Spanish people, so I understand that it should be in their native language. It seems to me a more than reasonable change.

Best regards.